backend/de.metas.edi/src/main/java/de/metas/edi/api/impl/DesadvBL.java

```diff
@@ -293,6 +298,14 @@ private I_EDI_DesadvLine retrieveOrCreateDesadvLine(
 		return newDesadvLine;
 	}
 
+	@Nullable
+	private BigDecimal getQtyOrdered_Override(@NonNull final I_C_OrderLine orderLineRecord)
+	{
+		final OrderLineId orderLineId = OrderLineId.ofRepoId(orderLineRecord.getC_OrderLine_ID());
+		final I_M_ShipmentSchedule shipSched = shipmentSchedulePA.getByOrderLineId(orderLineId);
+		return shipSched != null ? shipSched.getQtyOrdered_Override() : null;
```

Comment:
i think `shipSched.getQtyOrdered_Override()` will always return not null.
So in case of null will return zero.

i would suggest to also check InterfaceWrapperHelper.isNull(...)